packages/network/src/utils/propShapes.js

```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types';
+
+export const nodeStyleshape = PropTypes.shape({
```

Comment:
I don't think these are used? you could update to `nodeShape` and `linkShape`, though?